theano/gpuarray/extra_ops.py

```diff
@@ -456,6 +454,8 @@ def c_support_code_struct(self, node, nodename):
 @register_opt2([CumsumOp], 'fast_compile')
 def use_gpu_cumsumop(op, ctx_name, inputs, outputs):
     if inputs[0].dtype == 'float32':
+        if isinstance(inputs[0].type, GpuArrayType):
+            return
```

Comment:
Yes. Applies for all the opts that is used by new global opt, will make that change